rust/worker/src/blockstore/types.rs

```diff
@@ -13,12 +13,25 @@ use thiserror::Error;
 pub(crate) enum BlockfileError {
     #[error("Key not found")]
     NotFoundError,
+    #[error("Invalid Key Type")]
+    InvalidKeyType,
+    #[error("Invalid Value Type")]
+    InvalidValueType,
+    #[error("Transaction already in progress")]
+    TransactionInProgress,
+    #[error("Transaction not in progress")]
+    TransactionNotInProgress,
 }
 
 impl ChromaError for BlockfileError {
     fn code(&self) -> ErrorCodes {
         match self {
-            BlockfileError::NotFoundError => ErrorCodes::InvalidArgument,
+            BlockfileError::NotFoundError
+            | BlockfileError::InvalidKeyType
+            | BlockfileError::InvalidValueType => ErrorCodes::InvalidArgument,
+            BlockfileError::TransactionInProgress | BlockfileError::TransactionNotInProgress => {
```

Comment:
nit could we format these the same?

Reply:
this is the rust fmt default, so just going to leave it